meta-lmp-base/recipes-kernel/linux/kmeta-linux-lmp-5.15.y.inc

```diff
@@ -1,4 +1,4 @@
 KERNEL_META_REPO ?= "git://github.com/foundriesio/lmp-kernel-cache.git"
 KERNEL_META_REPO_PROTOCOL ?= "https"
 KERNEL_META_BRANCH ?= "linux-v5.15.y"
-KERNEL_META_COMMIT ?= "0edf6afabb7b17c7de611af3f24af3f970f87200"
+KERNEL_META_COMMIT ?= "31a784cd8a9c62011993cd9918c63d3108dc1514"
```

Comment:
This is the hash for the master branch and not for linux-v5.15.y, which is the branch used by this recipe.

Change to 8203249516013fc2fd292da5d204d802808868c0 which is the same change but at the right branch.

Reply:
right, sorry for my confusion